postgresql/wal.xml

```diff
@@ -17,18 +17,19 @@
 
   <para>
    La fiabilité est une propriété importante de tout système de base de
-   données sérieux. <productname>PostgreSQL</productname> fait tout ce qui est
-   en son pouvoir pour garantir une fiabilité à toute épreuve. Un des aspects
-   de cette fiabilité est que toutes les données enregistrées par une
-   transaction
-   validée doivent être stockées dans un espace non volatile, un espace
-   non sensible aux coupures de courant, aux bogues du système d'exploitation
+   données sérieux. <productname>PostgreSQL</productname> fait tout
+   son possible pour garantir un fonctionnalité fiable. Un des aspects
```

Comment:
Je ne sais pas trop ce que tu voulais écrire mais "un fonctionnalité fiable" ne me paraît pas très français. Tu voulais dire quoi ?

Reply:
Oups!  « Un fonctionnement fable » (to guarantee reliable operation). Correctif poussé.